pkg/config/config.go

```diff
@@ -1045,6 +1045,8 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("serverless.logs_enabled", true)
 	config.BindEnvAndSetDefault("enhanced_metrics", true)
 	config.BindEnvAndSetDefault("capture_lambda_payload", false)
+	config.BindEnvAndSetDefault("trace_enabled", false)
+	config.BindEnvAndSetDefault("trace_managed_services", false)
```

Comment:
Would it make sense to add `serverless.` prefix to the config keys? 

Also, would it make sense to document this in `config_template.yaml`?